src/applications/personalization/profile360/containers/CopyMailingAddress.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import isEqual from 'lodash/isEqual';
+import pick from 'lodash/pick';
+
+import {
+  consolidateAddress
+} from '../../../../platform/forms/address/helpers';
+
+import {
+  FIELD_NAMES
+} from '../constants/vet360';
+
+import {
+  selectVet360Field,
+  selectEditedFormField
+} from '../selectors';
+
+import {
+  cleanAddressDataForUpdate
+} from '../actions/misc';
+
+class CopyMailingAddress extends React.Component {
+  onChange = (event) => {
+    event.stopPropagation();
+    if (event.target.checked) {
+      this.props.copyMailingAddress(
+        consolidateAddress(this.props.mailingAddress)
```

Comment:
@ncksllvn do we need consolidateAddress here? When we call `copyMailingAddress`, which in turn calls `onChange`, that hits the `updateProfileFormField` action. In that action, `cleanAddressDataForUpdate` is automatically called